.circleci/config.yml

```diff
@@ -53,17 +53,34 @@ jobs:
             ./build-rdoc.sh
 
       - run:
-          name: Show diff
+          name: Check diff
           command: |
-            git --no-pager diff --exit-code
+            set +e
+            git add .
+            PATCH_FILE="<< pipeline.git.revision >>.patch"
+            git --no-pager diff --cached --exit-code > $PATCH_FILE
+            failed=$?
+
+            if [[ "$failed" == "1" ]]; then
+              cat $PATCH_FILE
+              MLFLOW_REPO_ID=136202695
+              PATCH_FILE_URL="https://${CIRCLE_BUILD_NUM}-${MLFLOW_REPO_ID}-gh.circle-artifacts.com/0/${PATCH_FILE}"
+              echo "========== Run the following commands to apply the diff above =========="
+              echo "PATCH_FILE_URL=\"$PATCH_FILE_URL\""
+              echo 'REDIRECTED_URL=$(curl -Ls -o /dev/null -w %{url_effective} $PATCH_FILE_URL)'
+              echo 'curl $REDIRECTED_URL | git apply --verbose'
+              echo "========================================================================"
```

Comment:
This allow us to quickly apply the diff by simply copy & pasting the commands in the terminal.

Reply:
The output looks like this:

```
% PATCH_FILE_URL="https://8927-136202695-gh.circle-artifacts.com/0/21ed7fa02120377bd091db2277396603c26d1180.patch"
$ REDIRECTED_URL=$(curl -Ls -o /dev/null -w %{url_effective} $PATCH_FILE_URL)
$ curl $REDIRECTED_URL | git apply --verbose
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  2118  100  2118    0     0   2097      0  0:00:01  0:00:01 --:--:--  2097
Checking patch mlflow/R/mlflow/man/mlflow_create_registered_model.Rd...
Checking patch mlflow/R/mlflow/man/mlflow_delete_registered_model.Rd...
Applied patch mlflow/R/mlflow/man/mlflow_create_registered_model.Rd cleanly.
Applied patch mlflow/R/mlflow/man/mlflow_delete_registered_model.Rd cleanly.
```

Reply:
I think this approach is simpler than:

1. Open the Artifacts tab on CricleCI
2. Open the patch there and download it
3. Run `git apply /path/to/patch`